gradio/components.py

```diff
@@ -931,15 +937,13 @@ def __init__(
     ):
         """
         Parameters:
-        default(str): IGNORED
+        default_value(str): IGNORED
```

Comment:
This is no longer ignored. We should say in the docstring something like, "default image showed by the input component. If None, no image is shown"

Reply:
Actually it currently is ignored and not used in the component. Let me implement this in an individual commit.

Reply:
not sure about how to handle default image input, opened an issue and added a TODO